### ISSUE TITLE
Creating WindowedContext from an existing winit::Window

### ISSUE THREAD

**Issue by notdanilo** [rust-windowing/glutin#1347]
Is there any way to create a `WindowedContext` from an existing `winit::Window`?

I am developing a GPU library which I plan to support both Vulkan and OpenGL as its backend. The only commonplace for creating their contexts is `winit`.

**Comment by notdanilo** [comment]
I just discovered about `RawContext`.

But even the example isn't working.

Running `cargo run --example raw_context` returns:
```
thread 'main' panicked at 'explicit panic', glutin/src/platform_impl/unix/x11.rs:157:34
```

**Comment by kchibisov** [comment]
Fixed in #1439.
